package.json

```diff
@@ -1,47 +1,18 @@
 {
   "name": "native-template",
-  "version": "6.0.0",
+  "version": "6.1.0",
   "private": true,
   "scripts": {
     "postinstall": "patch-package && jetify",
     "configure": "native-mobile-toolkit configure --config-path='./config.json' --verbose"
   },
   "dependencies": {
-    "@react-native-community/art": "1.2.0",
-    "@react-native-community/async-storage": "1.12.0",
-    "@react-native-community/cameraroll": "4.0.1",
-    "@react-native-community/datetimepicker": "3.0.4",
-    "@react-native-community/geolocation": "2.0.2",
-    "@react-native-community/masked-view": "0.1.10",
-    "@react-native-community/netinfo": "5.9.7",
-    "@react-native-community/push-notification-ios": "1.7.1",
-    "@react-native-firebase/app": "10.1.0",
-    "@react-native-firebase/messaging": "10.1.0",
     "react-native": "0.63.3",
-    "react-native-ble-plx": "2.0.1",
-    "react-native-calendar-events": "2.1.0",
-    "react-native-camera": "3.40.0",
     "react-native-code-push": "6.3.0",
-    "react-native-device-info": "6.1.0",
-    "react-native-fast-image": "8.3.2",
-    "react-native-geocoder": "0.5.0",
     "react-native-gesture-handler": "1.8.0",
-    "react-native-image-picker": "2.3.4",
```

Comment:
Image picker will also stay as we now have a patch for it

Reply:
I'd rebase then

Reply:
Done